tests/e2e/chain/config.go

```diff
@@ -158,6 +160,21 @@ func initGenesis(c *internalChain) error {
 	}
 	appGenState[banktypes.ModuleName] = bz
 
+	var govGenState govtypes.GenesisState
+	if err := util.Cdc.UnmarshalJSON(appGenState[govtypes.ModuleName], &govGenState); err != nil {
+		return err
+	}
+
+	govGenState.VotingParams = govtypes.VotingParams{
+		VotingPeriod: VotingPeriod,
+	}
+
+	gz, err := util.Cdc.MarshalJSON(&govGenState)
```

Comment:
So we unmarsal and then marshal back only to add `VotingPeriod`? Is it possible to instead add this flag when we are initializing genesis in the chain_init container?

Reply:
I see, I misinterpreted how it all works. Thanks for clarifying